index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+        <header>
+        <nav>
+        <div class="Main-container"><p>My Logo</p>
+        <div class="menu"><i class="fa fa-bars"></i></div>
```

Comment:
- [ ] It is required for your hamburger to be inside a `nav` tag. Kindly change 
`<div class="menu"><i class="fa fa-bars"></i></div>` 
  to
 `<nav class="menu"><i class="fa fa-bars"></i></nav>`

Reply:
it is already inside the nav